common/source/docs/common-lua-scripts.rst

```diff
@@ -103,12 +103,18 @@ A Location userdata object supports the following calls:
 
 - :code:`loiter_xtrack([should_xtrack])` - If the location is used for a loiter location this flags if the aircraft should track from the center point, or from the exit location of the loiter.
 
-- :code:`get_distance(other_loc)` - Given a Location this calculates the horizontal distance between the two locations in meters.
+- :code:`get_distance(other Location)` - Given a Location this calculates the horizontal distance between the two locations in meters.
```

Comment:
bad change, no spaces are possible for argument names, and this would read like it's actually 2 arguments.

Reply:
fixed